contents/blog/clickhouse-announcement.md

```diff
@@ -0,0 +1,32 @@
+---
+date: 2021-08-13
+title: We just made PostHog Open Source 1000x more scalable via ClickHouse
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: mo-shehu
+featuredImage: ../images/blog/filename.png
+featuredImageType: full
+---
+We're excited to announce a major under-the-hood upgrade for PostHog Open Source! Over the past few weeks, our team have been hard at work moving over from Postgres to ClickHouse, the same database that powers PostHog Cloud.
+
+This means that anyone can now get started using PostHog for free at scale - you'll no longer be limited to 10k monthly active users. Read on to find about more about the benefits and how we did this!
+
+## Why ClickHouse?
+
+ClickHouse is a database designed to enable fast, efficient analytics computations across multiple machines. We are moving [PostHog Open Source]() from PostgreSQL to ClickHouse because ClickHouse is a better overall experience on every level. Postgres runs out of capacity quickly and doesn’t support the advanced analytics we need for our product, and there is no way to scale PostgreSQL beyond a certain point.
```

Comment:
I'd tweak this second and third sentence to be a little clearer and also reduce repetition.

"We are moving [PostHog Open Source]() from PostgreSQL because it doesn't support some features we need and struggles to scale beyond 10k monthly active users. ClickHouse, on the other hand, offers all the functionality we need and offers a much better experience with large volumes of data."

to ClickHouse because ClickHouse is a better overall experience on every level. Postgres runs out of capacity quickly and doesn’t support the advanced analytics we need for our product, and there is no way to scale PostgreSQL beyond a certain point.